### ISSUE TITLE
Vertical tmux borders are dashed with the Cascadia Code font

### ISSUE THREAD

**Issue by antoineco** [mintty/mintty#1119]
The tweak from #628 to improve the rendering of vertical tmux pane separators doesn't seem to be effective with Cascadia Code. (The rendering is fine with other fonts)

This is not a big deal and I can live with it. In fact, I find the way terminal rows look slightly higher in mintty a bit more comfortable to read compared to Windows Terminal Preview. However, I was curious whether there was something mintty might not be handling properly. I remember this wouldn't be the first time Cascadia Code is causing some trouble: mintty/wsltty#257.

Here is my mintty configuration:

```ini
# %APPDATA%\Roaming\wsltty\config

Font=Cascadia Code
FontHeight=12
FontWeight=400
```

![image](https://user-images.githubusercontent.com/3299086/131036407-938f125b-775b-4a00-abfb-7a9d75235027.png)

Below is a comparison of mintty and Windows Terminal Preview side-by-side, using the same font (Cascadia Code, 12). Notice the "dashed" aspect in mintty:

![image](https://user-images.githubusercontent.com/3299086/131033283-f38448a2-fc01-4479-9e73-e8ffd7accb6c.png)

tmux uses the Unicode character `U+2502 : BOX DRAWINGS LIGHT VERTICAL {Videotex Mosaic DG 14}` as vertical separator.

The issue can be reproduced without tmux by executing the following command in a shell: `printf "│\n│\n│\n"`.

**Comment by mintty** [comment]
See also #935 and mintty/wsltty#353.

**Comment by mintty** [comment]
Released 3.7.5 with self-drawn box characters.
